system-variables.md

```diff
@@ -1004,6 +1004,17 @@ mysql> SHOW GLOBAL VARIABLES LIKE 'max_prepared_stmt_count';
 - Default value: (system dependent)
 - This variable shows the system time zone from when TiDB was first bootstrapped. See also [`time_zone`](#time_zone).
 
+### `tidb_accelerate_user_creation_update` <span class="version-mark">New in v9.0.0</span>
+
+- Scope: GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
+- Type: Boolean
+- Default value: `OFF`
+- When too many (for example 100K) SQL users are created in the database, operations such as create user/modify password/change user privilege would suffer severe performance impact. The reason is that every modification to the user cause reload of all users' data. Setting the variable to `ON` is best practice is such scenario.
+    * `0` or `OFF`: modify user related operations cause reloading of all users' data, the performance is not good.
+    * `1` or `ON`: modify user related operations only reload the active users, so the performance is better.
```

Comment:
```suggestion
    * `0` or `OFF`: When performing operations such as creating users, changing passwords, or setting permissions, each operation triggers a full reload of all user data into memory. This may result in poor performance in scenarios with a large number of users.
    * `1` or `ON`: 
    Creating users or performing password changes and permission settings for non-logged-in users will not trigger a full reload of all user data into memory.
    Modifying logged-in users will only reload the updated user data into memory.
    This provides better performance in scenarios with a large number of users.
```